packages/now-next/test/integration/index.test.js

```diff
@@ -274,7 +274,7 @@ it(
       buildResult: { output },
     } = await runBuildLambda(path.join(__dirname, 'serverless-config-object'));
 
-    expect(output['index.html']).toBeDefined();
+    expect(output['/']).toBeDefined();
```

Comment:
Does this work in production?

Reply:
Worked for the test deployment https://test-fkrbr4mnw.now.sh/

Reply:
I would have thought this would need to be `output['/index']`

Reply:
They don't usually have leading slashes at all...

Reply:
Oh yeah, I mean `output['index']` since these are the file outputs

Reply:
Updated

Reply:
Do you have a test deployment for this too?

Reply:
https://test-ct3ids2j6.now.sh/